server/plugin.go

```diff
@@ -96,36 +94,36 @@ func (p *Plugin) handleAuthorizeInADD(w http.ResponseWriter, r *http.Request) {
 	userID := r.URL.Query().Get("mattermost_user_id")
 
 	if userID == "" {
-		fmt.Println("Not authorized")
-		http.Error(w, "Not authorized", http.StatusUnauthorized)
+		p.API.LogError("Cannot authorize in ADD. Missing 'mattermost_user_id' param")
+		http.Error(w, "Cannot authorize in ADD. Missing 'mattermost_user_id' param.", http.StatusUnauthorized)
```

Comment:
A bit OT: I would be nice if `handleAuthorizeInADD` would return an `error` that a wrapper method then logs and returns via http.

Are you interested in including these changes into this PR?

Reply:
Good point! Done :+1: